packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -124,7 +124,8 @@ export const getDocs: () => JSX.Element = () => {
         solution. Using the provided APIs, you can enable the composite to support uploading files and displaying them
         on the send box before sending, and the message thread once they have been sent or received. For an end to end
         tutorial on enabling file sharing with Azure Blob Storage as your choice of a storage solution, refer to our
-        tutorial. https://docs.microsoft.com/en-us/azure/communication-services/tutorials/file-sharing-tutorial
+        [tutorial](https://docs.microsoft.com/azure/communication-services/tutorials/file-sharing-tutorial). File
+        sharing is supported for Teams interop scenarios by default.
```

Comment:
Should we be specific here and call out: File sharing from Teams to ACS is supported. And when the next feature is released just update it?